recipes-extended/xen/files/libxl-domain-state.patch

```diff
@@ -120,7 +120,7 @@ Index: xen-4.6.4/tools/libxl/libxl_utils.c
  
 +int libxl_update_state_direct(libxl_ctx *ctx, libxl_uuid xl_uuid, const char * state)
 +{
-+    char path[48];
++    char path[sizeof("/state/00000000-0000-0000-0000-000000000000/state")];
```

Comment:
This PR was already tested and merged in master, so I suggest that any cosmetic change to it should be done as a new PR. Will merge soon if that's ok.